2d-survival-template/project/main_level.tscn

```diff
@@ -1,11 +1,13 @@
-[gd_scene load_steps=15 format=3 uid="uid://ctg31nmbj8psi"]
+[gd_scene load_steps=18 format=4 uid="uid://ctg31nmbj8psi"]
 
 [ext_resource type="Script" path="res://ui.gd" id="1_7q265"]
 [ext_resource type="Script" path="res://player.gd" id="1_585jn"]
 [ext_resource type="Texture2D" uid="uid://caapuolmxiw4p" path="res://assets/art/greybox.png" id="1_eu058"]
+[ext_resource type="Script" path="res://addons/godot_sfxr/SfxrStreamPlayer.gd" id="4_74fmw"]
 [ext_resource type="Script" path="res://interactable.gd" id="4_ecb12"]
 [ext_resource type="PackedScene" uid="uid://b6adeviogiean" path="res://interactable.tscn" id="5_edi4b"]
 [ext_resource type="Script" path="res://enemy.gd" id="5_ghdyc"]
+[ext_resource type="Script" path="res://sfx_player.gd" id="7_bmho6"]
```

Comment:
@loteque I'd recommend moving the sfx player out of this scene. For this project, I'd make it global. Every scene can jsut have a global reference to SFX.
Otherwise, we may have to start thinking about whether scenes that get factored out still have accesss to it like then we factor out enemy, bush, or pond.

I think we wil want to factor out so you can actually design a level while I am prototyping mecchanics in a test scene.

Reply:
No nodes in a scene have references to sfxplayer. sfxplayer gets its references through values passed by the signal. 

We can sfxplayer global it shouldn't matter either way. 

if we were to make it global, I'd prefer if the flow of data was one way; i.e.: `signalbus -> sfxplayer` with the requirement that sfxplayer should not mutate objects passed to it through signals. The only thing shxplayer should do is listen on signals and watch values it cares about in objects that are passed to it through the signal and play sfx.

Reply:
...and done. https://github.com/loteque/2024_oh_jam/pull/1/commits/06988b053e04f1a2b6a095ce43546c9b440f87a6